config/rbac/byohost_editor_role.yaml

```
@@ -35,3 +35,14 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - ""
```

Looks like all the permission are also identical for `byohosts` apiGroups. We should combine them together like the following?
```suggestion
  - ["", "byohosts"]
```